api/proto/oap/opentelemetry/metrics.proto

```diff
@@ -0,0 +1,666 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
```

Comment:
Copy the whole file into erda is not a good choice.
We can use proto-go from https://github.com/open-telemetry/opentelemetry-proto-go and only register some necessary http/grpc method.